.clang-format

```diff
@@ -10,7 +11,67 @@ AlwaysBreakTemplateDeclarations: true
 AlwaysBreakAfterReturnType: All
 PointerAlignment: Left
 AllowShortIfStatementsOnASingleLine: false
-BreakBeforeBraces: Allman
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterClass:      true
+  AfterControlStatement: Always
+  AfterEnum:       true
+  AfterFunction:   true
+  AfterNamespace:  true
+  AfterObjCDeclaration: true
+  AfterStruct:     true
+  AfterUnion:      false
+  AfterExternBlock: true
+  BeforeCatch:     true
+  BeforeElse:      true
+  BeforeLambdaBody: false
+  IndentBraces:    true
+  BeforeWhile: false
+  IndentBraces: false
+  SplitEmptyFunction: true
+  SplitEmptyRecord: true
+  SplitEmptyNamespace: true
+
+# Disable formatting options which may break tests.
+SortIncludes: false
+ReflowComments: false
+
+# Indent preprocessor directives
+IndentPPDirectives: AfterHash
+
+#The following section is to properly format the onedpl_config.h file which is
```

Comment:
ignore from formatting or force it to c++ via file extension -> .hpp

in any case, delete the obj-c style specification

Reply:
I was able to remove the language specification in the clang-format style which resolves the issue without needing to have separate language specifications.  The header may be thought of as Objective-C internally to clang-format, but this doesn't cause problems, as the settings can be shared like this, and formatting is properly tracked and controlled for the file.  

It seems that renaming the header to `.hpp` extension would fix also it, but this fix seems to be better and less risky.

Reply:
Great!